### ISSUE TITLE
HABTM Native Functionality Not Valid

### ISSUE THREAD

**Issue by panbanda** [djezzzl/database_consistency#45]
I am getting this error for a basic model that has a standard HABTM association.  This requires me to manually specify the model instead of using the Rails built in methods.  

```
fail HABTM_Categories article_id column is required in the database but do not have presence validator
```

For example: 

```ruby
class Category < ApplicationRecord
  has_and_belongs_to_many :articles
end
class Article < ApplicationRecord
  has_and_belongs_to_many :categories
end
```

This fails because the join table has a `null: false` in the migration but the generated join model in Rails doesn't have the `validates :article_id, presence: true` validation.

Suggestion: Maybe ignore habtm methods used in rails as its automatically generated?

**Comment by djezzzl** [comment]
Hey @panbanda, 

For now, I'd suggest you to disable this check for this particular model using our configuration.

Am I right that you don't have model `ArticlesCategory`, if so, then you're right, we probably should skip that check at all. Or maybe, you can create that model even if it will be almost empty, and add those validations there.

**Comment by panbanda** [comment]
Yes I don't have that model in my source as its behind the scenes magic.  I didn't know I could disable a check for a particular model.  I'll have to look that up thanks.

**Comment by djezzzl** [comment]
Yes, you can 👍 

https://github.com/djezzzl/database_consistency/blob/master/rails-example/.database_consistency.yml

**Comment by djezzzl** [comment]
Hey,

I've just released a fix for configuration checker (0.7.7). Please have a look on it and the FAQ (https://github.com/djezzzl/database_consistency/blob/master/FAQ.md). 

You can now disable that model from checks. 

Let me know if it doesn't help you. Have a great day 👍
